docs/docs/upgrading/migrate_v5_v6.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 1
+---
+
+# Upgrading to ICS v6.x from v5.x
+
+ICS specific changes are outlined below.
+
+Pre-requisite version for this upgrade: any from the `v5.x` release line.
+
+## Note
+
+## Provider
+
+### Migration (v5 -> v6)
+
+ConensusVersion was bumped to `6`.
+
+### Governance proposals
+
+#### Consumer addition proposal
+
+```shell
+interchain-security-pd tx gov submit-proposal <proposal_file.json>
+```
+
+####  Consumer removal proposal
```

Comment:
Not sure if we need all 3 sections on addition, modification, and removal since they use the exact same command.